doc/ref_model/chapters/chapter08-annex.md

```diff
@@ -6,232 +6,133 @@
-<li>glance-api</li>
-<li>glance-registry</li>
-<li>glance-rabbitmq</li>
-<li>heat-api</li>
-<li>heat-cfn</li>
-<li>heat-engine</li>
-<li>heat-rabbitmq</li>
-<li>keystone-api</li>
-<li>keystone-rabbitmq</li>
-<li>keystone-rabbitmq</li>
-</ul>
-</li>
-<li>Validate maridb cluster is insync
-<ul>
-<li>Studown mariadb and upon restart ensure its sync up with masterdb.</li>
-<li>Maria DB is single point of failure</li>
-</ul>
-</li>
-<li>Validate ceph
-<ul>
-<li>Restart ceph-osd</li>
-<li>Document VNF impact while ceph is unavailable and once ceph service is restoredbeing restored.</li>
-</ul>
-</li>
-</ul>
-</li>
-<ul>
-<li><span style="text-decoration: underline;"><strong>Security - see Ch 7 for complete list</strong></span>
-<ul>
-<li>Validation above is performed using both RBAC Roles and User group policies, both of Admin and User Roles.
-</ul>
-<li>**SUMMARY:** Validate User Role to ensure it allow user to perform all designated task and prohibits user performing any unassigned task.</li>
-<li>Validate Security Policy/Rules are enforced</li>
-</ul>
-</li>
-</ul>
-</ul>
+**FuncTest**
+
+FuncTest test suites must be run as part of NFVi validation against all OpenStack based reference implementations. At a minimum, FuncTest executions will include execution of the functest-smoke and functional-benchmarking test suites.
+
+Additional Resources for FuncTest:
+- [Project Description](https://wiki.opnfv.org/display/functest/Opnfv+Functional+Testing#OpnfvFunctionalTesting-Testcases)
+- [User Guide](https://opnfv-functest.readthedocs.io/en/stable-hunter/testing/user/userguide/index.html)
+- [Test Suite Overview](https://opnfv-functest.readthedocs.io/en/stable-hunter/testing/user/userguide/test_overview.html)
+
+Example, or Reference, Functest Status Reporting Artifacts:
+- [Rally Verification](http://artifacts.opnfv.org/functest/functest-opnfv-functest-smoke-iruya-tempest_full-run-259/results/tempest_full/tempest-report.html)
+- [Rally Tasks](http://artifacts.opnfv.org/functest/functest-opnfv-functest-benchmarking-iruya-rally_full-run-169/results/rally_full/rally_full.html)
+
+**Baremetal**
+**SUMMARY:** Control and compute node hardware, bios, firmware, network interfaces, and base operating system configuration must be validated.
+- Interface - Validate nic status for all member in bond1 group
+- Interface - MTU speed for bond1 interface
+- Grub - SR-IOV is enabled
+- Numa - Validate numa configuration on each node
+- Numa - Validate total memory available is distributed equally across numa boundaries
+- VFs - Validate VFs are availablebeing created on each PCI SYS Interface.
+- Modules - Validate the following modules are loaded
+  - bonding
+  - 8021q
+  - i40e
+- OS: IPTables are enabled.
+- OS: ntp is enabled and configured to sync system time
+- OS: Dedicate mount point for following file system
+  - /var/log
+  - /var/lib/nova
+- OS: Validate Huge Pages are configured and enabled based on target infrastructure. (not available for all flavours)
```

Comment:
Huge pages are not mandatory for the basic profile. See nfvi.com.cfg.004 in [5.2.1 Virtual Compute](https://github.com/cntt-n/CNTT/blob/599f92486c190e46231ff7a25ead31a4fc2ba008/doc/ref_model/chapters/chapter05.md#521-virtual-compute).

Reply:
This test case has a caveat of "based on target infrastructure" and goes further to include "not available for all flavours". Does this already satisfy your comment?

Reply:
Yes, I guess. It was not totally clear what is the meaning of these. Who/what decides if this is executed or not?

Reply:
This test case is just host config validation. As you point out, 5.2.1 describes multiple configurations where huge pages available for basic, network, and compute profiles. If the RI implements huge pages to support these features, we would expect a test case to validate they are configured upon deploy of the NFVi.